README-SNAP.md

```diff
@@ -56,9 +67,15 @@ Erlang settings are stored in the `/var/snap/couchdb/current/etc/vm.args` file.
 configuration tool can be used to quickly change the node name and security cookie:
 
 ```bash
-$ sudo snap set couchdb name=couchdb@1.2.3.4 setcookie=cutter
+$ sudo snap set couchdb name=couchdb@1.2.3.4 setcookie=$COOKIE
 ```
 
+Where COOKIE is an enviroment variable. You can auto generated a cookie with the command 
+below. 
+
+```bash
+$ export COOKIE=`echo $RANDOM | md5sum | head -c 20; echo;`
```

Comment:
This seems to work in `bash` though in other shells like `zsh` it seems to always generate the same value which might be dangerous:

```
% export COOKIE=`echo $RANDOM | md5sum | head -c 20; echo;`
% echo $COOKIE
dc8b85e421d60214e049
% export COOKIE=`echo $RANDOM | md5sum | head -c 20; echo;`
% echo $COOKIE
dc8b85e421d60214e049
```

In couchdb-pkg we use something like:

```
echo $(dd if=/dev/random bs=1 count=38 status=none | base64 | tr -cd '[:alnum:]')
9qiRUeH4SMYCNoP8emESCrhMrQtf6q0EKv98s6McMfUvS33r7Y
```

Reply:
Ok. I modified to use `/dev/random` to bake a better cookie